### PR TITLE
Make release updates

### PR DESCRIPTION
**Pull request by matsl**
## What

Make release updates

## Why

Maybe something like this could be used. I have not included generation of the webpage yet. Not fully tested either since I don't want to start tagging the release and do ftp uploads and mocking these steps will still not fully verify things will work.

What do you think of trying out this together and see where things break? We can then see what makes sense to add to the process and what is best to have as separate steps?

### CODE REVIEW

**Comment by rswgnu** [comment]
Sure, sounds good.  -- Bob

File: Makefile [hunk 1]
```diff
@@ -392,16 +389,6 @@ git-pull:
 git-push:
 	git push
 
-# Once the release version number is updated in hyperbole.el and the release is pushed to
```
Comment (rswgnu) [inline Makefile:395]: Please leave these comments somewhere in the Makefile so we know that Elpa builds are automatic.

**Review by rswgnu — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -377,30 +374,26 @@ website:
...
+# Generate and distribute a Hyperbole release to ftp.gnu.org.
 # One step in this is to generate an autoloads file for the Koutliner, kotl/kotl-autoloads.el.
-release: package git-push $(pkg_dir)/hyperbole-$(HYPB_VERSION).tar.gz elpa ftp
-	@ echo; echo "Hyperbole $(HYPB_VERSION) released to elpa and ftp.gnu.org successfully."
+release: package $(pkg_dir)/hyperbole-$(HYPB_VERSION).tar.gz ftp git-tag-release
+	@ echo; echo "Hyperbole $(HYPB_VERSION) released to ftp.gnu.org successfully."
 
 # Ensure local hyperbole directory is synchronized with master before building a release.
 git-pull:
+	echo "If this step failes check your work directory for not commited changes"
```
Comment (rswgnu) [inline Makefile:386]: Replace with 'fails' and 'committed'.

**Review by rswgnu — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (matsl) [reply Makefile:395]: Added it at the top of the Makefile. Is that OK ro do you want it closer to the pkg and release targets?

Comment (rswgnu) [reply Makefile:395]: Is ok.

**Review by rswgnu — APPROVED** [review]

**Pull request merged**